packages/api/tests/lambdas/test-kinesis-consumer.js

```diff
@@ -226,7 +226,7 @@ test.serial('An SNS Fallback retry, should throw an error if message is invalid
 
   await t.throwsAsync(
     handler(snsEvent, {}, testCallback),
-    { message: 'Unexpected end of JSON input' }
+    { message: "Expected property name or '}' in JSON at position 1" }
```

Comment:
is this an anecdotal diagnosis of what this error will be? it seems more specific than the previous, and that concerns me

Reply:
It's the result of updates to the JSON interpreter in Node 20 - it's checking precisely what the error will be given the input -- in that sense it's no more (or less) specific than the previous test, it's the error is more specific. 

e.g. '{' isn't valid JSON because it's missing a closing tag.   Original error said "unexpected end to input", new error says "there's no "}" in the last char of input".      

There's a reasonable assertion here that the problem is we're not trapping all JSON errors and throwing our own generic/aggregate error, but that at least feels solidly out of scope.

Reply:
ok, makes sense